nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -67,7 +70,9 @@ in {
       };
 
       extraGSettingsOverridePackages = mkOption {
-        default = [];
+        default = [] ++ [
+          pkgs.gnome3.gnome-shell
```

Comment:
Won’t the default value get overridden?

Reply:
Ah yes, will add a `cp` to `nixos-gsettings-desktop-schemas`